dp/cloud/docker/python/test_runner/entrypoint.sh

```diff
@@ -2,6 +2,7 @@
 set -e
 
 if [ "$TESTS_DEV" = true ] ; then
+    echo "This container will sleep infinitely when TESTS_DEV env is set to true, it's for tests development"
```

Comment:
Is it still deployed then?

Reply:
Yes, it's deployed with `dev_integration_tests` make target, it gives oportunity to enter the container, and run tests manually. I added this comment cause the container with just sleep can confuse a user

Reply:
Oh, Ok I think I misunderstood. I removed NMS in this PR. This change which you commented out was made by the way, and is not related with NMS disabling